File: 1-two_sum/1-two_sum.py
class Solution:
    def twoSum(self, nums: list[int], target: int) -> list[int]:
        nums_length = len(nums)
        sum_index = []

        for i in range(nums_length - 1):
            for j in range(i + 1,nums_length):
                if((nums[i] + nums[j]) == target):
                    sum_index.append(i)
                    sum_index.append(j)
                    return sum_index
        
        return sum_index

File: 1-two_sum/1-two_sum.cpp
#include <vector>

class Solution {
public:
    std::vector<int> twoSum(std::vector<int>& nums, int target) {
        std::vector<int> sum_indices;
        
        for(int i = 0; i < nums.size() - 1; i++) {
            for (int j = i + 1; j < nums.size(); j++) {
                if(nums.at(i) + nums.at(j) == target) {
                    sum_indices.push_back(i);
                    sum_indices.push_back(j);
                }
            }
        }

        return sum_indices;
    }
};